vortex-duckdb/src/scan.rs

```diff
@@ -54,17 +54,16 @@ impl std::fmt::Debug for VortexBindData {
 }
 
 pub struct VortexGlobalData {
-    file_paths: SegQueue<PathBuf>,
-    is_first_file_processed: AtomicBool,
-    /// We currently use a conversion cache to cache converted arrays, this id is used to
-    /// ensure that each cache created has a unique id used to name those arrays
-    conversion_cache_id: AtomicU64,
-    filter_expr: Option<ExprRef>,
-    projection_expr: ExprRef,
+    multi_scan: MultiScan,
+    cache_id: AtomicU64,
 }
 
 pub struct VortexLocalData {
-    exporter: Option<ArrayIteratorExporter>,
+    multi_scan_iterator: MultiScanIterator,
+    exporter: Option<ArrayExporter>,
+
+    // TODO(Alex): replace with global conversion cache
+    conversion_cache: ConversionCache,
```

Comment:
Just remove this. We know we need to plumb it through MultiScan as data attached to an individual scan task.

Reply:
I need some cache to live somewhere for the time being as this is passed to 
```
                local_state.exporter = Some(ArrayExporter::try_new(
                    &array_result?.to_struct()?,
                    &mut local_state.conversion_cache,
                )?);
```
Just passing the newly created cache directly to `&mut local_state.conversion_cache` crashes.